docs/sources/documentation/pipeline_operators/decomposition/FastICA.md

```diff
@@ -0,0 +1,60 @@
+# Fast algorithm for Independent Component Analysis
+* * *
+
+Uses Scikit-learn's FastICA to transform the feature set.
+
+## Dependencies
+    sklearn.decomposition.FastICA
+
+
+Parameters
+----------
+    input_df: pandas.DataFrame {n_samples, n_features+['class', 'group', 'guess']}
+        Input DataFrame to scale
+    n_components: int
+            The number of components to keep
+    tol: float
+        Tolerance on update at each iteration.
+
+Returns
+-------
+    modified_df: pandas.DataFrame {n_samples, n_components + ['guess', 'group', 'class']}
+        Returns a DataFrame containing the transformed features
+
+
+Example Exported Code
+---------------------
+
+```Python
+import numpy as np
+import pandas as pd
+from sklearn.cross_validation import StratifiedShuffleSplit
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.decomposition import FastICA
+
+# NOTE: Make sure that the class is labeled 'class' in the data file
+tpot_data = pd.read_csv('PATH/TO/DATA/FILE', sep='COLUMN_SEPARATOR')
+training_indices, testing_indices = next(iter(StratifiedShuffleSplit(tpot_data['class'].values, n_iter=1, train_size=0.75, test_size=0.25)))
```

Comment:
All of the docs need to be updated with the new way we implement the cross-validation split. We use `train_test_split` now, e.g.,

```python
training_indices, testing_indices = train_test_split(tpot_data.index, stratify=tpot_data['class'].values, train_size=0.75, test_size=0.25)
```